<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>16</width>
    <height>9</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>16</horstretch>
       <verstretch>9</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>16</width>
       <height>9</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>주문목록</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6" rowstretch="1,2,2" columnstretch="2">
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>판매 순위</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8" columnstretch="2,3">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView_3"/>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="labelChart">
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>판매 내역</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QTableView" name="tableView_2">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>매출</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="3,1">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout" rowstretch="10,1">
         <item row="0" column="0">
          <widget class="QTableView" name="tableView_1">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelSum">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="20,2,1">
         <item row="2" column="1">
          <widget class="QPushButton" name="set_btn">
           <property name="text">
            <string>설정</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="labelStatus">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="reset_btn">
           <property name="text">
            <string>초기화</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCalendarWidget" name="calendar"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
